assets/release/release-aarch64.json

```diff
@@ -1,20 +1,20 @@
 {
   "release": {
-    "base": "4.13.0-0.nightly-arm64-2023-01-21-233324"
+    "base": "4.13.0-0.nightly-arm64-2023-01-17-171314"
```

Comment:
Could you use images from `last_rebase.sh`? To have this changes in sync with what we already have